app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h2> This is a test </h2>
\ No newline at end of file
+<section >
+  <%= render'partials/user', user: @user %>
+    <% @posts.each do |post| %>
+   <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+    <ul class="post-list">
+    <%= render 'partials/post', post: post %></ul>
+    </a>
+    </ul>
+    <% end %>
+  <button type='button'>Pagination</button>
+</section>
```

Comment:
- [ ] Kindly adjust your posts list to be centered on the page. It shifts to the left disproportionately.

- [ ] Kindly move the pagination button to be centered and at the bottom of the page.

  See [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABY4RVEMXJTGGPFXUAVZGZ6AY4KOFSA)

  ![image](https://user-images.githubusercontent.com/54848363/205383132-0419fe52-bd94-411d-bbb3-7614aec67cec.png)